webrtc.html

```diff
@@ -1204,16 +1242,23 @@
             media state if it results in different connectivity being
             established.</p>
 
-            <p>If the candidate parameter is malformed, throw a
-            <code>SyntaxError</code> exception and abort these steps.</p>
-
-            <p>If the candidate is successfully applied, the user agent MUST
-            queue a task to invoke <var>successCallback</var>.</p>
+            <p>When <code>addIceCandidate</code> is invoked, <a>queue an
+            RTCPeerConnection task</a> to process the incoming candidate:</p>
 
-            <p>If the candidate could not be successfully applied, the user
-            agent MUST queue a task to invoke <var>failureCallback</var> with a
-            <code>DOMError</code> object whose <code>name</code> attribute has
-            the value TBD (TODO InvalidCandidate and InvalidMidIndex).</p>
+            <ol>
+              <li>If the candidate parameter is malformed, throw
+              a <code>SyntaxError</code> exception and abort these steps.</li>
```

Comment:
We can't throw from a queued task. It has to be done as a direct result of a method being called.